libs/sdk-core/src/breez_services.rs

```diff
@@ -360,8 +360,8 @@ impl BreezServices {
             .map_err(|err| anyhow!(err))
     }
 
-    /// Fetch a specific payment by its hash.
-    pub async fn payment_by_hash(&self, hash: String) -> Result<Option<Payment>> {
+    /// Get a specific payment by its hash.
+    pub async fn get_payment_by_hash(&self, hash: String) -> Result<Option<Payment>> {
```

Comment:
For rust it is better to leave `payment_by_hash`. In general we are not using the `get` notation in the sdk so let's be consistent.